client/my-sites/people/team-list/test/team.jsx

```diff
@@ -0,0 +1,75 @@
+/**
+ * External dependencies
+ */
+
+import React from 'react';
+import { shallow } from 'enzyme';
+import { expect } from 'chai';
+import sinon from 'sinon';
+
+/**
+ * Internal dependencies
+ */
+import useFakeDom from 'test/helpers/use-fake-dom';
+import useMockery from 'test/helpers/use-mockery';
+
+describe( 'Team', function() {
+	let PeopleListSectionHeader,
+		ReactInjection,
+		Team,
+		translations,
+		translate,
+		configMock;
+
+	useFakeDom();
+	useMockery( mockery => {
+		configMock = sinon.stub();
+		configMock.isEnabled = sinon.stub();
+		mockery.registerMock( 'config', configMock );
+	} );
+
+	before( function() {
+		ReactInjection = require( 'react/lib/ReactInjection' );
+		translations = [];
+		translate = function() {
+			translations.push( Array.from( arguments ) );
+		};
+		ReactInjection.Class.injectMixin( { translate: translate } );
```

Comment:
!hanks @gziolo! I'm actually thinking of just leaving this PR as a spike and reimplementing from scratch (writing tests first).

My intention with the mock translation function was to add assertions for all calls.